src/components/CommonButton.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+import '../styles/CommonButton.css';
+
+export default function CommonButton({
+  children,
+  onClick,
+  onSubmit,
+  className,
```

Comment:
그리고, 기본값을  빈 스트링으로 주어 값이 전달되지 않았을 때 의도치않게 undefined 등이 들어가지 않도록 처리해주시면 좋을 것 같아요!
```suggestion
  className = "",
```